quint/src/generated/Effect.g4

```diff
@@ -1,12 +1,12 @@
-/**
+/*,*
  * A grammar of Quint Effects
  *
  * @author: Gabriela Moreira, Informal Systems, 2022
  */
 grammar Effect;
 
 effect:   concrete                                           # concreteEffect
-        | '(' (effect (', ' effect)*)? ')' '=>' effect       # arrowEffect
+        | '(' (effect (',' effect)*)? ')' '=>' effect       # arrowEffect
```

Comment:
Allows effect variables to be written without spaces after commas. E.g., we can now write `Read[r0,r1]` as well as `Read[r0, r1]`. Previously, only the latter was allowed.